lib/generators/phlex/install/install_generator.rb

```diff
@@ -55,6 +54,21 @@ def create_application_layout
 
 		def create_application_view
 			template "application_view.rb", Rails.root.join("app/views/application_view.rb")
+		end
+
+				private
+
+		def tailwind_configuration_path
+			@_tailwind_configuration_path ||=
+				Pathname.new(tailwind_configuration_files.first)
+		end
+
+		def tailwind_configuration_files
+			Dir.glob([
+				"#{Rails.root}/tailwind.config.js",
+				"#{Rails.root}/app/**/tailwind.config.js",
+				"#{Rails.root}/config/**/tailwind.config.js",
+						])
```

Comment:
Yet another questionable Rubocop thingamajig 😵‍💫

Not sure how to handle these (now and in the future)?

Reply:
Just do it the way you feel is right and we’ll adjust the Rubocop configuration accordingly.